### PR TITLE
recwatch: Fix watching newly created files on macOS

### PR DESCRIPTION
**Pull request by aequitas**
Fixes: https://github.com/purpleidea/mgmt/issues/33

### CODE REVIEW

File: test/util.sh [hunk 1]
```diff
@@ -11,9 +11,11 @@ export MGMT="$ROOT/mgmt"
 if [[ $(uname) == "Darwin" ]] ; then
 	export timeout="gtimeout"
 	export mktemp="gmktemp"
+	export STAT="gstat"
```
Comment (purpleidea) [inline test/util.sh]: how come some of these are uppercase, and others are lowercase? what is correct?

File: docs/quick-start-guide.md [hunk 2]
```diff
@@ -134,10 +134,6 @@ discouraged either). Meaning it might work but in the case it doesn't you would
 have to provide your own patches to fix problems (the project maintainer and
 community are glad to assist where needed).
 
-There are currently some issues that make `mgmt` less suitable to run for
```
Comment (purpleidea) [inline docs/quick-start-guide.md]: Why kill this? No systemd support or equivalent on osx...

File: test/util.sh (hunk 1 above)
Comment (aequitas) [reply test/util.sh]: If it is not capitalised it hits: https://github.com/koalaman/shellcheck/wiki/SC2154

Preferably all shell scripts should be conform shellcheck but thats a branch I still need to finish.

File: docs/quick-start-guide.md (hunk 2 above)
Comment (aequitas) [reply docs/quick-start-guide.md]: True, I just couldn't find any issue to reference :)

**Comment by purpleidea** [comment]
Cools! Merged!

**Pull request closed without merging**